src/table/table.scss

```diff
@@ -102,6 +102,7 @@
   // Sort icon
   .iui-sort {
     visibility: hidden;
+    display: flex;
```

Comment:
Not sure it changed anything..

Reply:
It fixes #274

Reply:
why we could not repro that on our side..?

Reply:
This can actually be removed, because we are applying this same rule on all svgs.

As for the repro, that issue has buggy code, so I left a comment there